src/Lean/Data/Lsp/Extra.lean

```diff
@@ -34,6 +34,15 @@ instance : FromJson WaitForDiagnostics :=
 instance : ToJson WaitForDiagnostics :=
   ⟨fun o => mkObj []⟩
 
+structure LeanFileProgressProcessingInfo where
```

Comment:
Could you add an entry for `$/lean/fileProgress` in the module docstring?

Reply:
Done.